requirements.txt

```diff
@@ -1,3 +1,4 @@
+attrs==18.2.0
```

Comment:
@jacklatourette What is attrs doing here?

Reply:
attrs needed to be upgraded to be compatible the pytest version being used in circle ci

Reply:
@jacklatourette Ok then please move it to `requirements-dev.txt` where dev related packages shold reside.